transition.js

```diff
@@ -0,0 +1,16 @@
+class TransitionScreen {
```

Comment:
is this supposed to be blank?
seems odd to have something thats presumeably getting created as a entity and added to the gameEngine (since it has update/draw methods) but does nothing....

so I assume then that this transition object is just not finished atm?